themes/custom/droopler_theme/src/components/d-p-banner/d-p-banner.twig

```diff
@@ -41,7 +41,11 @@
         {% block content_header %}
         {% endblock %}
 
+        {% set has_content_header = block('content_header')|render|striptags|trim is not empty %}
```

Comment:
Why do not close in this IF statement the `block content_header` as well?
Also, make sure that indentations are correct.